### ISSUE TITLE
For each object ... generate fixtures

### ISSUE THREAD

**Issue by spinx** [nelmio/alice#321]
I'd like to generate one address for each user. 

Adding references obviously generates multiple rows for the same user, which I don't want. 

Ideally, something like this: 

```
DomainBundle\Entity\UserAddress (foreach @user*):
    user_address:
        user_address_id (unique): <numberBetween()>
```

This would generate one address per user. 

I can try to add this, if you deem it useful. 

Or is there another way to achieve the same thing ?

**Comment by spinx** [comment]
So I figured out a way to do a basic base with <current()> but i think it would still be valuable to support this for complex cases.

**Comment by reinfi** [comment]
isn't this https://github.com/nelmio/alice/pull/315 what you are looking for?

**Comment by theofidry** [comment]
Closed via #315
